Gemfile

```diff
@@ -60,7 +60,7 @@ group :development, :test do
   gem 'rspec-rails', '~> 3.3'
   gem 'thin'
   gem 'bullet'
-  gem 'mailcatcher', '0.6.3'
```

Comment:
I did a quick smoke test on OS X and Docker, and I was able to see the sent emails. If you come across an issue, we can always open a new PR that reverts the version number with an explanation so we can remember what the issue was in the future.